uni/ulsp/Makefile

```diff
@@ -0,0 +1,59 @@
+BASE = ../..
+include $(BASE)/Makedefs.uni
+
+progs=launch-lsp
+src=launch-lsp.icn workspace.icn database.icn server.icn completion.icn
+default=$(progs)
+
+export IPATH=$(UNI)/unidoc
+# export IPATH=/home/mark/unicon-lspfork/unicon/uni/unidoc
+
+launch-lsp:launch-lsp.icn workspace.u server.u database.u completion.u
+	unicon -u launch-lsp.icn
+
+workspace.u:workspace.icn
+	unicon -u -c workspace.icn
+
+database.u:database.icn
+	unicon -u -c database.icn
+
+server.u:server.icn database.u completion.u workspace.u
+	unicon -u -c server.icn
+
+completion.u:completion.icn
+	unicon -u -c completion.icn
+
+# all:
+# 	$(UNICON) $(DASHG) -o launch-lsp database.u workspace.u server.u completion.u
+
+clean:
+	rm -f *.u launch-lsp uniclass*
+
+
+# BASE = ../..
+# include $(BASE)/Makedefs.uni
+
+# progs=launch-lsp
+# src=launch-lsp.icn workspace.icn server.icn database.icn completion.icn
+# default=$(progs)
+
+# export IPATH=$(UNI)/unidoc
+# # export IPATH=/home/mark/unicon-lspfork/unicon/uni/unidoc
+
+# launch-lsp:launch-lsp.icn workspace.u server.u database.u completion.u
+# 	unicon -u launch-lsp.icn
+
+# workspace.u:workspace.icn
+# 	unicon -u -c workspace.icn
+
+# server.u:server.icn
+# 	unicon -u -c server.icn
+
+# database.u:database.icn
+# 	unicon -u -c database.icn
+
+# completion.u: completion.icn
+# 	unicon -u -c completion.icn
+
+# clean:
+# 	rm -f *.u launch-lsp uniclass*
```

Comment:
Should we remove this?

Reply:
Removed comments
Related commit: https://github.com/uniconproject/unicon/pull/305/commits/3354f020e827e3eee006f7ef02998c24b8b67048